Doc/library/concurrent.futures.rst

```diff
@@ -54,11 +54,16 @@ Executor Objects
        specified by setting *chunksize* to a positive integer. For very long
        iterables, using a large value for *chunksize* can significantly improve
        performance compared to the default size of 1. With :class:`ThreadPoolExecutor`,
-       *chunksize* has no effect.
+       *chunksize* has no effect. By default, a reasonable number of tasks are
+       queued beyond the number of workers, an explicit *prefetch* count may be
+       provided to specify how many extra tasks should be queued.
 
        .. versionchanged:: 3.5
           Added the *chunksize* argument.
 
+       .. versionchanged:: 3.7
```

Comment:
If this is to be merged, the `versionchanged` should be updated (to 3.7.1 or 3.8 or whatever version it actually goes in).